packages/card-validator/index.ts

```diff
@@ -107,16 +107,23 @@ export function validateCVC(
   cvc: string,
   cardNumber: string
```

Comment:
We need to mark the cardNumber arg as optional now I think

Reply:
Done, and tests added 👍 Thanks!